apps/src/templates/manageStudents/manageStudentsClient.js

```diff
@@ -0,0 +1,137 @@
+import {gql, makeVar, useMutation, useReactiveVar} from '@apollo/client';
```

Comment:
Note to self: This file is where a lot of the interesting stuff is, including the custom hook and editing logic.